nav2_controller/dwb_critics/src/oscillation.cpp

```diff
@@ -91,7 +94,8 @@ void OscillationCritic::onInit()
   oscillation_reset_dist_ = nav_2d_utils::searchAndGetParam(nh_, "oscillation_reset_dist", 0.05);
   oscillation_reset_dist_sq_ = oscillation_reset_dist_ * oscillation_reset_dist_;
   oscillation_reset_angle_ = nav_2d_utils::searchAndGetParam(nh_, "oscillation_reset_angle", 0.2);
-  oscillation_reset_time_ = nav_2d_utils::searchAndGetParam(nh_, "oscillation_reset_time", -1.0);
+  oscillation_reset_time_ = nav2_util::durationFromSeconds(
+    nav_2d_utils::searchAndGetParam(nh_, "oscillation_reset_time", -1.0));
```

Comment:
Should we pull this functionality out of nav_2_util to our nav2_util package? I think a lot of other code would duplicate it otherwise.

Reply:
You mean the `nav_2d_utils` package? For now I'd prefer to leave that where it is because it is part of the DWB repository. The plan is to eventually use that repo and remove all this code from ours.